Jcaccamo/readme phase2 qml scenes

jared-esri opened the pull request:
@ldanzinger It's so close I can almost taste it. Scenes Category is done for QML and ready for review. Thanks!

I also came across some mistakes here and there on the CPP ones I already did and made the corrections as I came across them.

ldanzinger commented on ArcGISRuntimeSDKQt_QMLSamples/Scenes/AddIntegratedMeshLayer/README.md:
```diff
@@ -4,16 +4,34 @@ View an integrated mesh layer from a scene service.
-2. Create an IntegratedMeshLayer with the URL to an integrated mesh layer scene service.
-3. Add the layer to the scene's operational layers collection.
+
+1. Create an `Scene`.
```
```suggestion
1. Create a `Scene`.
```

ldanzinger commented on ArcGISRuntimeSDKQt_QMLSamples/Scenes/Animate3DSymbols/README.md:
```diff
@@ -18,46 +21,36 @@ Camera Controls (Top Right Corner):
+5. Add heading, pitch, and roll attributes to the graphic. Get the attributes from the graphic with `Graphic::attributes`.
+6. Create a `SimpleRenderer` object and set its expression properties.
+7. Add graphic and a renderer to the graphics overlay.
+8. Create a `OrbitGeoElementCameraController` which is set to target the graphic.
```
```suggestion
8. Create an `OrbitGeoElementCameraController` which is set to target the graphic.
```

ldanzinger commented on ArcGISRuntimeSDKQt_QMLSamples/Scenes/BasicSceneView/README.md:
```diff
@@ -1,15 +1,30 @@
 ## How it works
 
-This is the most basic sample for displaying a scene. It shows how to create a scene view, and add in a scene that contains a surface.
+1. Create an `Scene` object with the `BasemapImagery` basemap.
```
```suggestion
1. Create a `Scene` object with the `BasemapImagery` basemap.
```

ldanzinger commented on ArcGISRuntimeSDKQt_QMLSamples/Scenes/ChangeAtmosphereEffect/README.md:
```diff
@@ -10,15 +10,14 @@ Select one of the three available atmosphere effects. The sky will change to dis
-
-1. Create a ``Scene`` and display it in a ``SceneView``.
-2. Change the atmosphere effect with property ``atmosphereEffect``.
+1. Create a `Scene and display it in a `SceneQuickView.
```
tick marks are off here

ldanzinger commented on ArcGISRuntimeSDKQt_QMLSamples/Scenes/OrbitCameraAroundObject/README.md:
```diff
@@ -1,72 +1,71 @@
+2. Load a plane graphic and set the `OrbitGeoElementCameraController` to be the targetGeoElement
+3. Assign this controller to the `cameraController` sceneview property to set the camera to the scene view  
+4. Set the heading, pitch and distance camera properties :
+ * `OrbitGeoElementCameraController::cameraHeadingOffset`
```
all of these properties are using :: instead of .

ldanzinger commented on ArcGISRuntimeSDKQt_QMLSamples/Scenes/SyncMapViewSceneView/README.md:
```diff
@@ -1,25 +1,36 @@
-- ``GeoView::currentViewpoint``
-- ``GeoView::setViewpoint``
+* GeoView
+* GetCurrentViewPoint
```
I don't believe this is what it is called in QML (no get prefix). please double check

ldanzinger commented on ArcGISRuntimeSDKQt_QMLSamples/Scenes/SyncMapViewSceneView/README.md:
```diff
@@ -1,25 +1,36 @@
+* MapView
+* viewpointChanged
+* SceneView
+* SetViewPoint
```
```suggestion
* setViewpoint
```

ldanzinger commented on ArcGISRuntimeSDKQt_QMLSamples/Scenes/TerrainExaggeration/README.md:
```diff
@@ -1,22 +1,32 @@
-- `Surface.elevationExaggeration`
+
+* Scene
+* Scene::baseSurface
```
```suggestion
* Scene.baseSurface
```

ldanzinger commented on ArcGISRuntimeSDKQt_QMLSamples/Scenes/TerrainExaggeration/README.md:
```diff
@@ -1,22 +1,32 @@
+* Scene
+* Scene::baseSurface
+* Surface
+* Surface::elevationExaggeration
```
```suggestion
* Surface.elevationExaggeration
```

jared-esri commented:
@ldanzinger So I addressed your comments. Then I realized they were in the CPP as well and initially just made the changes here. Decided I would put those in their own PR so I reverted that commit. In any case, these should be good now.
